### ISSUE TITLE
Error creating Application Customizer named 'Application customizer' using generator v1.3.4

### ISSUE THREAD

**Issue by waldekmastykarz** [SharePoint/sp-dev-docs#1000]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
Be able to create a project with Application Customizer.

#### Observed Behavior
When trying to bundle project with an Application Customizer named _Application customizer_ created using generator v1.3.4 the following error occurs:

```
Error - Unknown
 JSON validation failed:
/Users/clippy/tmp/spfx_134_2/config/serve.json

Error: #/serveConfigurations (Specify a set of configurations that ...)
       Additional properties not allowed:
```

#### Steps to Reproduce
- create new project for Online only
- add Application Customizer
- Use _Application customizer_ as the name
- `gulp bundle`
- 💥

**Comment by manishgarg1** [comment]
@waldekmastykarz , thank you for reporting the issue. We are able to repro it and are trying to get it fixed quickly.

**Comment by andrewconnell** [comment]
Just tested & confirmed this is resolved in SPFx v1.7.1
